stdlib/Pkg/src/API.jl

```diff
@@ -430,7 +430,7 @@ function clone(url::String, name::String = "")
 end
 
 function dir(pkg::String, paths::AbstractString...)
-    @warn "`Pkg.dir(pkgname, paths...)` is deprecated; instead, do `import $pkg; abspath($pkg, \"..\", \"..\", paths...)`." maxlog=1
+    @warn "`Pkg.dir(pkgname, paths...)` is deprecated; instead, do `import $pkg; joinpath(dirname(abspath($pkg)), \"..\", paths...)`." maxlog=1
```

Comment:
With the `dirname` call you don't want the `..` anymore.

Reply:
You still need `..`, because `dirname` is the `src` directory, and `Pkg.dir` returns the parent directory.

Reply:
Ah, gotcha.